tests/modules/create.py

```diff
@@ -1,55 +1,47 @@
 import os
 
 import pytest
-import requests_mock
 
 import nf_core.modules
-from tests.utils import mock_api_calls
 
 
 def test_modules_create_succeed(self):
     """Succeed at creating the TrimGalore! module"""
-    with requests_mock.Mocker() as mock:
-        mock_api_calls(mock, "trim-galore", "0.6.7")
-        module_create = nf_core.modules.ModuleCreate(
-            self.pipeline_dir, "trimgalore", "@author", "process_single", True, True, conda_name="trim-galore"
-        )
-        module_create.create()
+    # FIXME This should use the mock?
+    module_create = nf_core.modules.ModuleCreate(
+        self.pipeline_dir, "trimgalore", "@author", "process_single", True, True, conda_name="trim-galore"
+    )
+    module_create.create()
```

Comment:
Is your suggestion to remove these mocks? As we are only testing the code here, mocking the calls to anaconda and biopython makes the tests faster and doesn't affect them.

Reply:
No, I'm sorry, I should have added that I realized last night I removed all the mocks, after I stepped away from the computer. Why I did that is hidden in the commit summary:
```
Responses actually throws an error if all the mocks aren't used. That
allowed me to clean up a few tests that weren't actually using the mocks
and find a few.
```
So I'll dig into this when I get some time, why those mocks aren't getting used properly.